engo_glfw.go

```diff
@@ -217,20 +224,44 @@ func RunPreparation(defaultScene Scene) {
 	WorldBounds.Max = Point{Width(), Height()}
 
 	SetScene(defaultScene, false)
+	game = defaultScene
+}
+
+func closeEvent() {
+	log.Println("Got close event")
+	game.Exit()
```

Comment:
https://github.com/EngoEngine/engo/blob/master/scene.go#L10

And a `sceneWrapper` contains a `scene Scene`.